### ISSUE TITLE
URL in Upgrade Message is Invalid

### ISSUE THREAD

**Issue by JonathanHourany** [dnote/dnote#421]
## Environment

* project (cli, server, browser-extension): 
CLi
* version: 
0.10.0
* operating system: 
Linux Mint

## Description
Used Dnote today and was given a prompt asking if I wanted to check for an upgrade. When I chose yes the following link message and link was printed to screen
```
    • to upgrade, see https://github.com/dnote/dnote/pkg/cli/blob/master/README.md
```
However, the link is invalid as it seems Github no longer structures paths in the web UI like this. I believe the line that needs to be update is here

https://github.com/dnote/dnote/blob/b2da22acae2433d61abf4b7b8e6d228f5f09c07e/pkg/cli/upgrade/upgrade.go#L107

I the link should just point to the top-level README.md now as the READM.md in `pkg/cli` doesn't any any information in it, but I'm not familiar enough with this project to create a PR right from the get go.

**Comment by sungwoncho** [comment]
> I the link should just point to the top-level README.md

That is a great idea.

> I'm not familiar enough with this project to create a PR right from the get go.

It's fine, your contribution is welcome. You can simply change the URL in https://github.com/dnote/dnote/blob/b2da22acae2433d61abf4b7b8e6d228f5f09c07e/pkg/cli/upgrade/upgrade.go#L107.

**Comment by sungwoncho** [comment]
Fixed in [v0.11.1](https://github.com/dnote/dnote/releases/tag/cli-v0.11.1). Thanks for the report.
